packages/core/test/eventDescriptors.test.ts

```diff
@@ -251,11 +251,43 @@ describe('event descriptors', () => {
         .transition(undefined, { type: 'event.foo.bar.first.second' })
         .matches('success')
     ).toBeFalsy();
+
+    expect(console.warn).toMatchMockCallsInlineSnapshot(`
```

Comment:
For example, this output shows that we log **three** things for a single (!) `machine.transition` call. This should likely be improved, it seems to be too excessive.